### PR TITLE
Add table of training material sources

### PR DESCRIPTION
**Pull request by mlehtiva**
As requested by customer in weekly user meeting

View here: https://csc-guide-preview.rahtiapp.fi/origin/mlehtiva-training-material-source-table/support/training-material/#training-materials-and-sources-from-csc-and-partners

### CODE REVIEW

File: docs/support/training-material.md [hunk 1]
```diff
@@ -51,3 +51,20 @@ for self study.
...
+| **[CSC Notebooks](https://notebooks.rahtiapp.fi)**                |  Introduction to Python, <br>Introduction to R, <br> 2h intro to Quantum computing  |
+| **[VideoCSC](https://video.csc.fi)**          | [Introductions to topics](https://video.csc.fi/category/Training%3EIntroductions+to+Topics/455249), <br> Recordings of weekly user zoom talks |
+| **[CSC YouTube-channel](https://www.youtube.com/c/CscFi)**                      | See also [Chipster channel](https://www.youtube.com/channel/UCnL-Lx5gGlW01OkskZL7JEQ), <br> [Elmer channel](https://www.youtube.com/user/elmerfem)
+| **[CSC Training GitHub](https://github.com/csc-training)**         | Repositories to course materials (~~some~~ many are old) |
+|**[LUMI Training events](https://www.lumi-supercomputer.eu/events/)**     | [Workshop on quantum computing](https://www.lumi-supercomputer.eu/events/workshop-on-quantum-computing-hybrid-systems/), <br> [LUMI-C environment](https://www.lumi-supercomputer.eu/events/detailed-introduction-to-the-lumi-c-environment-and-architecture/) |
+| **[CodeRefinery Lessons](https://coderefinery.org/lessons/)**       | Version control, <br> open and collaborative coding |
```
Comment (hdrei) [inline docs/support/training-material.md:67]:
```suggestion
| **[LUMI Training material](https://lumi-supercomputer.github.io/)**       | Material from past LUMI courses, <br> Technical documentation of LUMI software stacks , <br> LUMI software library |
| **Software guides for LUMI**       | [VASP](https://docs.lumi-supercomputer.eu/software/guides/vasp/), <br> [Quantum ESPRESSO](https://docs.lumi-supercomputer.eu/software/guides/quantumespresso/) , <br> [ParaView](https://docs.lumi-supercomputer.eu/software/guides/paraview/), <br> The list will grow in the future |
| **[CodeRefinery Lessons](https://coderefinery.org/lessons/)**       | Version control, <br> open and collaborative coding | 
```

File: docs/support/training-material.md [hunk 2]
```diff
@@ -51,3 +51,20 @@ for self study.
...
+| **[Training material collection](training-material.md) in docs.csc.fi (this page)** | Links to thematically collected material
+| **[CSC Notebooks](https://notebooks.rahtiapp.fi)**                |  Introduction to Python, <br>Introduction to R, <br> 2h intro to Quantum computing  |
+| **[VideoCSC](https://video.csc.fi)**          | [Introductions to topics](https://video.csc.fi/category/Training%3EIntroductions+to+Topics/455249), <br> Recordings of weekly user zoom talks |
+| **[CSC YouTube-channel](https://www.youtube.com/c/CscFi)**                      | See also [Chipster channel](https://www.youtube.com/channel/UCnL-Lx5gGlW01OkskZL7JEQ), <br> [Elmer channel](https://www.youtube.com/user/elmerfem)
+| **[CSC Training GitHub](https://github.com/csc-training)**         | Repositories to course materials (~~some~~ many are old) |
+|**[LUMI Training events](https://www.lumi-supercomputer.eu/events/)**     | [Workshop on quantum computing](https://www.lumi-supercomputer.eu/events/workshop-on-quantum-computing-hybrid-systems/), <br> [LUMI-C environment](https://www.lumi-supercomputer.eu/events/detailed-introduction-to-the-lumi-c-environment-and-architecture/) |
```
Comment (hdrei) [inline docs/support/training-material.md:66]:
```suggestion
| **[LUMI user guide](https://docs.lumi-supercomputer.eu/)**       | A getting started -guide and material to the use of LUMI |
|**[LUMI Training events](https://www.lumi-supercomputer.eu/events/)**     | [Workshop on quantum computing](https://www.lumi-supercomputer.eu/events/workshop-on-quantum-computing-hybrid-systems/), <br> [LUMI-C environment](https://www.lumi-supercomputer.eu/events/detailed-introduction-to-the-lumi-c-environment-and-architecture/) |
```

Comment (hdrei) [reply docs/support/training-material.md:66]: Adding/suggesting some materials about LUMI

File: docs/support/training-material.md [hunk 3]
```diff
@@ -51,3 +51,22 @@ for self study.
...
+| **LUMI Training material](https://lumi-supercomputer.github.io/LUMI-training-materials/)**       | Material from past LUMI courses |
```
Comment (hdrei) [inline docs/support/training-material.md:67]:
```suggestion
```

File: docs/support/training-material.md [hunk 4]
```diff
@@ -51,3 +51,20 @@ for self study.
...
+| **[VideoCSC](https://video.csc.fi)**          | [Introductions to topics](https://video.csc.fi/category/Training%3EIntroductions+to+Topics/455249), <br> Recordings of weekly user zoom talks |
+| **[CSC YouTube-channel](https://www.youtube.com/c/CscFi)**                      | See also [Chipster channel](https://www.youtube.com/channel/UCnL-Lx5gGlW01OkskZL7JEQ), <br> [Elmer channel](https://www.youtube.com/user/elmerfem)
+| **[CSC Training GitHub](https://github.com/csc-training)**         | Repositories to course materials (~~some~~ many are old) |
+|**[LUMI Training events](https://www.lumi-supercomputer.eu/events/)**     | [Workshop on quantum computing](https://www.lumi-supercomputer.eu/events/workshop-on-quantum-computing-hybrid-systems/), <br> [LUMI-C environment](https://www.lumi-supercomputer.eu/events/detailed-introduction-to-the-lumi-c-environment-and-architecture/) |
+| **[CodeRefinery Lessons](https://coderefinery.org/lessons/)**       | Version control, <br> open and collaborative coding | 
+| **[Aalto Python for Scientific Computing](https://scicomp.aalto.fi/training/scip/python-for-scicomp-2022/)**  | pandas, <br> plotting, <br> binder, <br> parallel programming |
```
[the remark this answers is not in the document]
Comment (mlehtiva) [reply docs/support/training-material.md:69]: I suppose the example were on purpose on different rows, to make it easier to read. 
Simplified "~~some~~ many are old" -> "some are old".

[the remark this answers is not in the document]
Comment (hdrei) [reply docs/support/training-material.md:69]: Ok. The second column of the last row is "a matter of opinion" what looks best. I would maybe get rid of the commas, if the words listed are put on different rows, but this is not any strong opinion

**Comment by joonas-somero** [comment]
> The content looks great! Maybe also add a `what's-new` entry to promote this?

I added a preliminary enrty, but it still needs to be filled out properly.

**Review by attesillanpaa — APPROVED** [review]

**Pull request merged**